### PR TITLE
Map 192 add filter reporting location fullstack

### PR DESCRIPTION
**Pull request by vasylneshchadimov**

### CODE REVIEW

File: frontend/src/components/Profile/BasicTabs.tsx [hunk 1]
```diff
@@ -85,6 +86,13 @@ export default function BasicTabs({
               {...a11yProps(3)}
             />
           )}
+          {(role === 'moderator' || role === 'admin') && (
```
Comment (dobbi-free) [inline frontend/src/components/Profile/BasicTabs.tsx:90]: I think it should be in one block. You should wrapp this moderation tabs in fragment

**Review by dobbi-free — APPROVED** [review]

File: backend/controllers/LocationsController.ts [hunk 2]
```diff
@@ -73,20 +73,20 @@ const LocationsController = {
...
+      //         return (
+      //           el === l.verificationStatus ||
+      //           l.verificationStatus === 'waiting'
+      //         );
+      //       } else {
+      //         return el === l.verificationStatus;
+      //       }
+      //     });
+      //   });
+      // }
```
Comment (obozhko-dev) [inline backend/controllers/LocationsController.ts]: Remove commented code, pls

**Review by obozhko-dev — CHANGES_REQUESTED** [review]
1.  maybe the best way use title of tab in one row how it did in another tabs ?
image

2. I think not bad add horizontal scroll to tabs , because in smallest screens we have a problem (maybe the problem has another causes, for example fixed width of tabs panel or our profile component, if it is - you can do nothing). Example of scrollable tabs here https://mui.com/material-ui/react-tabs/ in "Scrollable tabs", to do than you need add property variant="scrollable" and specify scrollButtons
image

File: backend/controllers/LocationsController.ts (hunk 2 above)
Comment (vasylneshchadimov) [reply backend/controllers/LocationsController.ts]: done

**Review by Yojuks — APPROVED** [review]

**Review by obozhko-dev — APPROVED** [review]

**Pull request merged**